src/sql/platform/connection/common/connectionConfig.ts

```diff
@@ -16,9 +16,9 @@ import { deepClone } from 'vs/base/common/objects';
 
 export const GROUPS_CONFIG_KEY = 'datasource.connectionGroups';
 export const CONNECTIONS_CONFIG_KEY = 'datasource.connections';
-export const CONNECTIONS_SORT_BY_CONFIG_KEY = 'datasource.connections.sortBy';
+export const CONNECTIONS_SORT_BY_CONFIG_KEY = 'datasource.connectionsSortOrder';
```

Comment:
if the name is sortOrder, I would expect the values being 'Ascending', 'Descending'. Is there something wrong with SortBy?

Reply:
I'm just going off what VS Code has already. Better to be consistent in this case I think

![image](https://user-images.githubusercontent.com/28519865/116583621-64eccc80-a8cb-11eb-92ea-63802cc9d162.png)